src/Hipace.cpp

```diff
@@ -139,9 +139,6 @@ Hipace::Hipace () :
         solver == "explicit",
         "hipace.bxby_solver must be predictor-corrector or explicit");
     if (solver == "explicit") m_explicit = true;
-    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
-        !(m_explicit && !m_multi_plasma.AllSpeciesNeutralizeBackground()),
-        "Ion motion with explicit solver is not implemented, need to use neutralize_background");
```

Comment:
Since we decided to merge this quickly and not add the two species within this PR, could you put this one back in please?